sale_delivery_split_date/models/stock_picking.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2018 Alex Comba - Agile Business Group
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class StockPicking(models.Model):
+
+    _inherit = "stock.picking"
+
+    min_dt = fields.Date(
+        string="Scheduled Date (for filter purpose only)",
+        compute="_compute_min_dt",
+        store=True,
+    )
+
+    @api.depends("scheduled_date")
+    def _compute_min_dt(self):
+        for picking in self:
+            min_dt = fields.Date.from_string(picking.scheduled_date)
```

Comment:
Please remove `from_string` because it is deprecated: 
>  kept for backwards compatibility, but consider `from_string` as deprecated, will probably
    be removed after V12 

(from https://github.com/odoo/odoo/blob/de81cc386d3a7f85b192a60b19477fcec8097624/odoo/fields.py#L1811-L1813)

In the latest versions (after `12.0` I think) the value of Date/Datetime fields are already `date`/`datetime` objects respectively, is this conversion still really needed?